meson.build

```diff
@@ -57,6 +58,10 @@ if shellcheck.found()
   test('shellcheck toolbox (deprecated)', shellcheck, args: [toolbox_sh])
 endif
 
+if codespell.found()
+  test('codespell toolbox', codespell, args: ['--skip', meson.build_root(), meson.source_root()])
```

Comment:
It should be `meson.project_build_root()`, not `meson.build_root()`.  Otherwise I get this when trying to build this locally:
```
meson.build:62: WARNING: Project targeting '>= 0.58.0' but tried
    to use feature deprecated since '0.56.0': meson.build_root. use
    meson.project_build_root() or meson.global_build_root() instead.
```

I don't know why the tests didn't fail.  We use `meson --fatal-meson-warnings ...` in them.

Reply:
> I don't know why the tests didn't fail. We use `meson --fatal-meson-warnings ...`
> in them.

They didn't fail because we aren't running the codespell test in our CI.  You need to add codespell to `playbooks/dependencies.yaml`.

Reply:
All the changes have been made and the pull request updated @debarshiray